### PR TITLE
Use new menu for Profile

### PR DESCRIPTION
**Pull request by haileyok**
This converts the Profile dropdown menu to the new dropdown. Also adding a `minWidth` of 170 to the dropdown on web, since some menus appear very small if there is only a few items/those items have a short description.

### CODE REVIEW

File: src/view/com/profile/ProfileMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+                        }
+                        onPress={onPressBlockAccount}>
+                        <Menu.ItemText>
+                          {profile.viewer?.blocking ? (
+                            <Trans>Unblock Account</Trans>
+                          ) : (
+                            <Trans>Block Account</Trans>
+                          )}
+                        </Menu.ItemText>
+                        <Menu.ItemIcon icon={Block} />
```
Comment (haileyok) [inline src/view/com/profile/ProfileMenu.tsx:250]: Suggestions for icons here would be nice. Can't find something that works well for both block _and_ unblock. Maybe something in the mod PR will be nice?

File: src/view/com/profile/ProfileMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+                />
+              </TouchableOpacity>
+            )
+          }}
+        </Menu.Trigger>
+
+        <Menu.Outer>
+          <Menu.Group>
+            <Menu.Item
+              testID="profileShareBtn"
```
Comment (estrattonbailey) [inline src/view/com/profile/ProfileMenu.tsx:188]: Was `profileHeaderDropdownShareBtn`, should check if this affects e2e

File: src/components/Menu/index.web.tsx [hunk 3]
```diff
@@ -125,6 +125,7 @@ export function Outer({children}: React.PropsWithChildren<{}>) {
             a.p_xs,
             t.name === 'light' ? t.atoms.bg : t.atoms.bg_contrast_25,
             t.atoms.shadow_md,
+            {minWidth: 170},
```
[the remark this answers is not in the document]
Comment (estrattonbailey) [reply src/components/Menu/index.web.tsx:128]: Could also allow a `style` prop here on web for cases where we do want this.

**Review by estrattonbailey — APPROVED** [review]
LGTM, will have a look at icons quick

File: src/view/com/profile/ProfileMenu.tsx (hunk 2 above)
Comment (haileyok) [reply src/view/com/profile/ProfileMenu.tsx]: oops, nice find.

**Comment by mozzius** [comment]
Added a fix for #2028

File: src/components/Menu/index.tsx [hunk 4]
```diff
@@ -68,7 +68,9 @@ export function Trigger({children, label}: TriggerProps) {
   })
 }
 
-export function Outer({children}: React.PropsWithChildren<{}>) {
+export function Outer({
+  children,
+}: React.PropsWithChildren<{style?: ViewStyle | ViewStyle[]}>) {
```
Comment (estrattonbailey) [inline src/components/Menu/index.tsx:73]: You can use `StyleProp<ViewStyle>`

Comment (haileyok) [reply src/components/Menu/index.tsx:73]: 🤯

**Comment by haileyok** [comment]
@mozzius Did you want to keep that follow/unfollow button there? I think it's fine but just making sure.

**Pull request merged**